charts/argo-workflows/README.md

```diff
@@ -357,6 +358,7 @@ Fields to note:
 [BackendConfigSpec]: https://cloud.google.com/kubernetes-engine/docs/concepts/backendconfig#backendconfigspec_v1beta1_cloudgooglecom
 [FrontendConfigSpec]: https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-features#configuring_ingress_features_through_frontendconfig_parameters
 [links]: https://argoproj.github.io/argo-workflows/links/
+[columns]: https://github.com/argoproj/argo-workflows/pull/10693
```

Comment:
where do you use this ref? 👀

Reply:
Oh it's here. This does not give users ref, not like README, so I wonder this helps users.
<img width="681" alt="image" src="https://github.com/argoproj/argo-helm/assets/30188755/c9bd7b6e-7ba3-47ae-b198-4b9370e1ad11">

Reply:
my bad, most of this is just a copy of the way `links` works, and I forgot to change the link on L137. I've updated it to point to the columns ref